manifests/components/grafana.jsonnet

```diff
@@ -0,0 +1,151 @@
+/*
+ * Bitnami Kubernetes Production Runtime - A collection of services that makes it
+ * easy to run production workloads in Kubernetes.
+ *
+ * Copyright 2018 Bitnami
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+local kube = import "../lib/kube.libsonnet";
+local kubecfg = import "kubecfg.libsonnet";
+local utils = import "../lib/utils.libsonnet";
+
+local GRAFANA_IMAGE = "bitnami/grafana:5.3.4-r6";
+local GRAFANA_DATASOURCES_CONFIG = "/opt/bitnami/grafana/conf/provisioning/datasources";
+
+// TODO: add blackbox-exporter
+
+{
+  p:: "",
+  metadata:: {
+    metadata+: {
+      namespace: "kubeprod",
+    },
+  },
+
+  prometheus:: error "No Prometheus service",
+
+  // Default to "Admin". See http://docs.grafana.org/permissions/overview/ for
```

Comment:
I think we should better explain why are choosing `-#2` from below two reasonable choices:
-#1  Use `GF_SECURITY_ADMIN_USER` / `GF_SECURITY_ADMIN_PASSWORD`
   to explicitly set an `admin`,  then set `auto_role:: "Editor"`
-#2  All Org users are `Admin`